float64 winch_pos
float64 rudder_pos
